parts/linux/cloud-init/artifacts/cse_install.sh

```diff
@@ -351,7 +350,8 @@ cleanUpImages() {
         if [[ $exit_code != 0 ]]; then
             exit $exit_code
         elif [[ "${images_to_delete}" != "" ]]; then
-            for image in "${images_to_delete[@]}"
+            images=$(${images_to_delete}) #alternatively use IFS+read but we are using bash anyways so this works too
```

Comment:
I believe you are looking for images=(${images_to_delete})
some local tests:
root@aks-nodepool1-34284155-vmss000006:/#             images=(${images_to_delete})
root@aks-nodepool1-34284155-vmss000006:/#             for image in "${images[@]}"
>             do
>                 echo ${image}
>             done
mcr.microsoft.com/oss/kubernetes/hyperkube:v1.18.10-hotfix.20210118
mcr.microsoft.com/oss/kubernetes/hyperkube:v1.17.16
mcr.microsoft.com/oss/kubernetes/hyperkube:v1.17.13
mcr.microsoft.com/oss/kubernetes/hyperkube:v1.17.11-hotfix.20200901.1
mcr.microsoft.com/oss/kubernetes/hyperkube:v1.18.8-hotfix.20200924
mcr.microsoft.com/oss/kubernetes/hyperkube:v1.16.15-hotfix.20200903
mcr.microsoft.com/oss/kubernetes/hyperkube:v1.17.11-hotfix.20200901
mcr.microsoft.com/oss/kubernetes/hyperkube:v1.17.9-hotfix.20200824.1
mcr.microsoft.com/oss/kubernetes/hyperkube:v1.16.10-hotfix.20200824.1
mcr.microsoft.com/oss/kubernetes/hyperkube:v1.16.13-hotfix.20200824.1
mcr.microsoft.com/oss/kubernetes/hyperkube:v1.17.7-hotfix.20200714.2
mcr.microsoft.com/oss/kubernetes/hyperkube:v1.18.6-hotfix.20200723.1
mcr.microsoft.com/oss/kubernetes/hyperkube:v1.18.4-hotfix.20200626.1
mcr.microsoft.com/oss/kubernetes/hyperkube:v1.17.3-hotfix.20200601.1
mcr.microsoft.com/oss/kubernetes/hyperkube:v1.16.9-hotfix.20200529.1
root@aks-nodepool1-34284155-vmss000006:/#